### ISSUE TITLE
Most Mods Not Working

### ISSUE THREAD

**Issue by BeansOnButter** [Sirius902/LuaBackend#24]
Most of my mods aren't working anymore, I'm using v1.8.0. It's been a while since I last used them, when they did work
I also tried using older versions but it's still not working for one reason or another

**Comment by cahudson94** [comment]
Did you update the mod files themselves? If the F2 console opens (which it should with mod files present on any version right now) then the hook from here is working as intended, the issue would then be with the scripts themselves.

**Comment by Ethereal-Batman** [comment]
Just wanted to add that as of this mornings game update through steam, all of my mods have ceased to function as well for KHFM1, Recom, and KHFM2. I am using v1.8.0 and the console does open when I hit F2, and the mods are detected in the scripts folder, but none of the mods will actually activate in their respective games.

**Comment by AghastShadow** [comment]
Came here to say the same exact thing as Ethereal. I even system restored because I thought it was a bad driver update. All the scripts load correctly but don't actually apply themselves in game. 

I've done complete reinstalls of everything as well just to make sure. I'm not sure if each individual script from the modders have to be updated again because of that random steam update or if this is a issue for here. 

For Example: I have a script that adds the save menu to KH1's inventory menu. It says it's initialized but it's not actually working in game. 

![image](https://github.com/user-attachments/assets/7be7827f-b94e-483e-b9d2-f25626c2ec48)

![image](https://github.com/user-attachments/assets/f809da5e-eac6-4ec2-afe5-b18a81668bf8)

**Comment by cahudson94** [comment]
This is due to the game update. It is not a problem with backend. Direct these conversations to the maintainers of the scripts as they will need to update the addresses behind them.

**Comment by AghastShadow** [comment]
Pardon my knowledge on these topics: Is this as easy as changing a line in the script?
I've let the modders know for a few of the ones I've downloaded but if it's possible to just change them myself easily than I rather do that instead waiting on each mod to get updated.

**Comment by cahudson94** [comment]
No the scripts operate off memory address information which we have to locate through programs that allow us to inspect the game as it is played. These addresses are what control everything in game and what we manipulate to make the scripts do what they do. If you have programming knowledge and want to look more into it you are welcome, but it is not as simple as "change this one line and it works"

**Comment by AghastShadow** [comment]
Thanks for taking the time out to explain it for me. I'll wait patiently for the mods to get updated if they do. They're just mostly quality of life stuff anyway ^^ I spent hours yesterday trying to figure it out on my own just to see if I could lol

**Comment by sourceAida** [comment]
Someone just made a fix and it works perfectly! https://steamcommunity.com/app/2552430/discussions/0/4763206964687861476/

**Comment by cahudson94** [comment]
To be clear they did not "make" it, that is down patching the exes to 1.0.0.9/1.0.0.1 versions which is known to work. This has the upside of no wait time on updates from our end and the downside of potential bug introduction depending on what portions of the exe were adjusted and if they are setup to interact with things outside the exe. Low chance of problems but still good to know. I am grateful to them as it will help people stay up and running though for sure.

This process is something that most of the modding discord channels advise people to do before updates anyway it is just info that is not highly disseminated.

**Comment by cahudson94** [comment]
At this time, a large volume of the mods within the community have gotten updates to work with the most recent patch. If you are still encountering issues and have downloaded the most recent version of your mod, please reach out to the individual maintainers of that mod and not this board as failing mods/crashing do not have to do with this hooking process.

**Comment by Sirius902** [comment]
Yup this is something that needs to be addressed on the individual mods' ends, the hooking process has been updated to work regardless of game version so the only thing left is for mod developers to update their mods to work for the newest release of the game.
